gen/src/write.rs

```diff
@@ -1270,6 +1270,13 @@ fn write_cxx_vector(out: &mut OutFile, vector_ty: &Type, element: &Ident, types:
     );
     writeln!(out, "  return s.size();");
     writeln!(out, "}}");
+    writeln!(
```

Comment:
Wondering whether I can or should generate src/cxx.h|cc from this. Need to look into that...